src/implementations/vanilla/components/tag/_tag.scss

```diff
@@ -47,6 +47,11 @@ $tag: null !default;
   }
 }
 
+// No wrap
+.ecl-tag--nowrap {
```

Comment:
but..from what i see we were doing this by default. But the question is: why? :)
I mean, i do see all the potential risks involved in this while i cannot really identify use cases where this might be useful, instead.
We are lowering this risk by setting this as a variant, as opt-in thingy but i'm not sure it still make sense even this way.

Reply:
I don't recall exactly, but I think that we added this by default for a reason, possibly coming from EWPP.
So it's probably safer to keep it as a variant for now